src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -359,7 +359,7 @@ void minimize_local_reorders(program& p, std::map<program_node*, format::type>&
             auto reorders_cnt = count_reorders(fmt_map, lo, node);
 
             if (reorders_cnt.number < best_reorder_cnt.number ||
-                (reorders_cnt.number == best_reorder_cnt.number && reorders_cnt.total_sizes < best_reorder_cnt.total_sizes) ) {
+                (reorders_cnt.number == best_reorder_cnt.number && reorders_cnt.total_sizes <= best_reorder_cnt.total_sizes)) {
```

Comment:
I think this condition is weird because it is updating the policy though total_size does not descrease :( Can we handle the problem in a more graceful way?

Reply:
Agree, PR is updated, the root cause is other points.